### PR TITLE
Module Detection Improvements

### PR DESCRIPTION
**Pull request by carlosfebres**
The goal here is closing #49.
- I created a script to update modules metadata (ABI & ByteCode mainly) automatically when the app builds.
- For every module, if the contract is a proxy, it reads the master copy and checks if it's master copy of a module.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,17 +38,26 @@
...
     "ganache-core": "2.12.1",
-    "husky": "^7.0.2",
     "prettier": "^2.3.1",
     "react-app-rewired": "^2.1.6",
     "react-scripts": "3.4.0",
-    "typescript": "~4.1.3"
+    "typescript": "~4.1.3",
+    "@zodiac-module/reality": "github:gnosis/zodiac-module-reality#main",
+    "@zodiac-module/exit": "github:gnosis/zodiac-module-exit#carlos/packages",
+    "@zodiac-module/delay": "github:gnosis/zodiac-modifier-delay#carlos/ethers-version",
```
Comment (carlosfebres) [inline package.json:47]: 🛑  We need to merge these PRs first:
- Delay Modifier: https://github.com/gnosis/zodiac-modifier-delay/pull/23
- Exit Module: https://github.com/gnosis/zodiac-module-exit/pull/44

Comment (auryn-macmillan) [reply package.json:47]: Done!

**Comment by auryn-macmillan** [comment]
Is this one good to go, @carlosfebres ?

**Comment by carlosfebres** [comment]
> Is this one good to go, @carlosfebres ?

@auryn-macmillan it is

**Comment by auryn-macmillan** [comment]
image
I notice that the cursor position is after the native token symbol. It's nit picky, but can we have the cursor show up in the correct place?

One way that might make this easier is to do it like this.
image

**Comment by samepant** [comment]
> image I notice that the cursor position is after the native token symbol. It's nit picky, but can we have the cursor show up in the correct place?
> 
> One way that might make this easier is to do it like this. image

this looks good to me, I think the separation makes sense

**Comment by carlosfebres** [comment]
@auryn-macmillan @samepant I applied the new design to the field:

Screen Shot 2021-11-14 at 11 22 39 PM

**Comment by auryn-macmillan** [comment]
That looks good to me.
However build appears to be failing on netlify and also locally for me.

**Comment by carlosfebres** [comment]
> That looks good to me. However build appears to be failing on netlify and also locally for me.

@auryn-macmillan just fixed the build. I think we can now merge this PR.

**Comment by carlosfebres** [comment]
@auryn-macmillan this PR is ready to get merged 🚀

**Review by auryn-macmillan — APPROVED** [review]
LGTM 🎉

**Pull request merged**